interface/.typesafe-i18n.json

```diff
@@ -1,5 +1,5 @@
 {
   "adapter": "react",
   "baseLocale": "pl",
```

Comment:
the baseLocale should be 'en', but this will only work when the rest of the index.ts translations file use the same format as was done for Polish with the none|single|many syntax

Reply:
This was already in dev, i did not change it.